app/views/landing.html

```diff
@@ -32,6 +32,10 @@ <h1 translate>Search for an event</h1>
     </div>
   </div>
 </div>
+<div class="text-center" ng-if="loadingMsg">
+  <h2>{{loadingMsg}}</h2>
```

Comment:
I think you can just hardcode `Searching Events&hellip;` here since the message never changes.

You can make it translatable by doing `<h2 translate>Searching Events&hellip;</h2>`.

Then `loadingMsg` could just be a flag (true/false). Can you think of a more descriptive variable name for it that implies it is for searching? Maybe `loadingSearchResults` or something.